inc/class-wpseo-meta.php

```diff
@@ -604,15 +604,15 @@ public static function validate_meta_robots_adv( $meta_value ) {
 	 *
 	 * @static
 	 *
-	 * @param  null   $null       Old, disregard.
+	 * @param  null   $check      The current status to allow updating metadata for the given type
```

Comment:
Could you change the variable type to `null|bool`.
Could you add a `.` at the end of the line (our coding standards are requiring this to be present).